test/ProgressOnderwijsUtils.Tests/MetaObjectBulkCopyTest.cs

```diff
@@ -152,6 +157,71 @@ public void BulkCopySupportsCumputedColumn()
             PAssert.That(() => fromDb.Computed);
         }
 
+        sealed class IncludingIdentityColumn : ValueBase<IncludingIdentityColumn>, IMetaObject, IPropertiesAreUsedImplicitly
+        {
+            public int Id { get; set; }
+            public int AnIdentity { get; set; }
+            public string Bla { get; set; }
+        }
+
+        sealed class ExcludingIdentityColumn : ValueBase<ExcludingIdentityColumn>, IMetaObject, IPropertiesAreUsedImplicitly
+        {
+            public int Id { get; set; }
+            public string Bla { get; set; }
+        }
+
+        [Fact]
+        public void BulkCopyIgnoresIdentityColumns()
+        {
+            var tableName = SQL($"#MyTable");
+            SQL($@"
+                create table {tableName} (
+                    Id int not null primary key
+                    , AnIdentity int not null identity(1,1) -- deliberately not placed at the end or start
+                    , Bla nvarchar(max) not null
+                )
+            ").ExecuteNonQuery(Context);
+
+            new[] {
+                new ExcludingIdentityColumn {
+                    Id = 11,
+                    Bla = "Something"
+                }
+            }.BulkCopyToSqlServer(Context, BulkInsertTarget.LoadFromTable(Context, tableName));
+
+            var fromDb = SQL($@"
+                select *
+                from {tableName}
+            ").ReadMetaObjects<IncludingIdentityColumn>(Context).Single();
+            PAssert.That(() => fromDb.AnIdentity == 1);
+        }
+
+        [Fact]
+        public void BulkCopySupportsKeepIdentity()

```

Comment:
Added test here for bulk inserting into tables with identities. with KeepIdentity.